### DISCUSSION TITLE
Pimcore portal engine installation error

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#14901] [Q&A / Support]
Hi ,

We purchased Pimcore's Enterprise version and attempted to install the Pimcore portal engine using the instructions provided below.
https://pimcore.com/docs/platform/Portal_Engine/
However, while installing the portal engine, we encountered the following error:

[ERROR] {"error":{"root_cause":[{"type":"security_exception","reason":"missing authentication credentials for REST
         request [/_alias/portalengine_filterdefinition]","header":{"WWW-Authenticate":["Basic realm="security"
         charset="UTF-8"","ApiKey"]}}],"type":"security_exception","reason":"missing authentication credentials for
         REST request [/_alias/portalengine_filterdefinition]","header":{"WWW-Authenticate":["Basic realm="security"
         charset="UTF-8"","ApiKey"]}},"status":401}

We would appreciate your assistance in resolving this matter.
The following is our system configuration for installing the portal engine.

Pimcore version: 10.5.20
Elastic search: Elastic 8.7
Enterprise Metadata Bundle
Statistics Explorer

**Comment by fashxp** [comment]
see https://github.com/pimcore/pimcore/discussions/14900
